### ISSUE TITLE
any functional working driver?

### ISSUE THREAD

**Issue by ftoledo** [FrescoLogic/FL2000#89]
I succefully compile for 5.10.x on debian 11 from other branches.
Anybody get a full working driver?

@LinJiabang do you still working for FrescoLogic?
This half implementation is useless

Why doesn't the vendor deliver a driver that is at least functional? even if you have bugs or it looks a bit bad... then the community can patch it to improve it.
But in these conditions it seems impossible.
I'll keep trying all the other forks in case I find something more functional.
it is a pity =(

**Comment by larrasket** [comment]
I don't think it will ever come to happen. I'm considering getting [this](https://www.amazon.com/Universal-Graphics-Displaylink-Supports-2048x1152/dp/B01CSG7TUC/) or [this](https://www.amazon.com/Universal-Graphics-Supports-displays-External/dp/B0943VNT67) instead since it provides a driver for linux.

**Comment by ftoledo** [comment]
> I don't think it will ever come to happen. I'm considering getting [this](https://www.amazon.com/Adapter-Multi-Display-Converter-Projector-Chromebook/dp/B0B7WB851B/ref=sr_1_15?crid=1SJWA2CCKRETR&keywords=usb%2Bto%2Bhdmi%2Blinux&qid=1681163241&sprefix=usb%2Bto%2Bhdmi%2Blinu%2Caps%2C308&sr=8-15&th=1) instead since it provides a driver for linux.

The description means that are not compatible with linux

 "Pero ten en cuenta: no es compatible con MAC/Linux/Vista/Chrome/Firestick. "

can you confirm if you run it on linux?
thanks!

**Comment by Nable80** [comment]
@ftoledo could you be more specific about the missing functions of this driver, please?

**Comment by larrasket** [comment]
@ftoledo I apologize I meant to put the url to WAVLINK chip instead I updated the comment. From the reviews I think yeah I can confirm it works with Linux.
